src/Pages/Auth/Components/Title.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+const Title = props => {
+  const { title } = props;
+  return <PageTitle>{title}</PageTitle>;
+};
```

Comment:
```suggestion
const Title = ({ title }) => {
  return <PageTitle>{title}</PageTitle>;
};
```
- 위와 같이 함수형 컴포넌트에서 props객체를 바로 구조 분해 할당해서 사용할 수 있습니다!